guide/src/main/scala/io/udash/web/guide/StatesToViewFactoryDef.scala

```diff
@@ -14,7 +15,7 @@ class StatesToViewFactoryDef extends ViewFactoryRegistry[RoutingState] {
       case RootState => RootViewFactory
       case ContentState => ContentViewFactory
 
-      case IntroState => IntroViewFactory
+      case IntroState => new MarkdownPageViewFactory[IntroState.type]()(Context.serverRpc.pages())
```

Comment:
apply?

Reply:
Also making in this way we constantly create new `ViewFactory` when we hit this `URI`. I don't know if it's intended or avoidable but recreating the entire view and presenter on `URI` hit does not seem right.

Reply:
Udash always creates a new `ViewFactory` (unless it's an object). We should recreate the view when we switch state, this one is not parameterized.